pos_with_tax/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: AGPLv3 License
+
+Point of sale with tax
+======================
+
+Include tax in price on POS.
+
+This addons takes care of tax configuration: price_include, showing prices
```

Comment:
Put it more at user level: `This addon shows the correct price indepently of the tax configuration (with or without the "Tax included in price" check).